adding firebase crashlytics demo project (#78)
codemagic-ci-cd/codemagic-sample-projects: integrations/firebase_crashlytics_demo_project/README.md

# firebase_crashlytics_demo_project

A new Flutter project.

## Getting Started

This project is a starting point for a Flutter application.

A few resources to get you started if this is your first Flutter project:

- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)

For help getting started with Flutter, view our
[online documentation](https://flutter.dev/docs), which offers tutorials,

> samples, guidance on mobile development, and a full API reference.

Reviewer: This looks like the generic Flutter README text. Use an example from one of the other demo projects to tell people what this project is, and to look at the codemagic.yaml

Author: Done!

# firebase crashlytics demo project

## Getting Started

In order to generate debug symbols, Firebase Crashlytics SDK must be installed using the following command line:

```
flutter pub add firebase_crashlytics
```

Or as an alternative it could be added in the **pubspec.yaml**:

```
dependencies:
  flutter:
    sdk: flutter
  firebase_crashlytics: ^2.5.2
```

> As soon as your build finishes successfully, debug symbols are generated, however if they need them to be present in the **Codemagic UI**, then the following path needs to be configured in **codemagic.yaml** under the artifacts section:

Reviewer: ```suggestion
As soon as your build finishes successfully, debug symbols are generated.  However, if you want them to be displayed in the **Codemagic UI** on the build page, then the following path needs to be configured in **codemagic.yaml** under the artifacts section:
```

Author: Done!

  firebase_crashlytics: ^2.5.2
```

> As soon as your build finishes successfully, debug symbols are generated.  However, if you want them to be displayed in the **Codemagic UI** on the build page, then the following path needs to be configured in **codemagic.yaml** under the artifacts section:

Reviewer: ```suggestion
As soon as your build finishes successfully, debug symbols are generated. However, if you want them to be displayed in the **Codemagic UI** on the build page, then the following path needs to be configured in **codemagic.yaml** under the artifacts section:
```
